data/json/items/comestibles/soup.json

```diff
@@ -51,89 +51,89 @@
     "type": "COMESTIBLE",
     "id": "soup_veggy",
     "name": "vegetable soup",
-    "weight": 125,
+    "weight": 254,
     "color": "red",
     "spoils_in": "5 days",
     "container": "can_food",
     "comestible_type": "DRINK",
     "symbol": "~",
     "quench": 8,
     "healthy": 1,
-    "calories": 200,
+    "calories": 84,
     "description": "A nutritious and delicious hearty vegetable soup.",
     "price": 400,
     "material": "veggy",
     "volume": 1,
     "phase": "liquid",
-    "charges": 4,
```

Comment:
Because you changed the charges of this item, you need to divide the weight and price by the number of charges.

In soup.json, you change charges on lines (other than this one):
68, 90, 112, 135, 180, 197, 210, 278, and 298

Reply:
for these the serving size didn't change, only the volume.